### ISSUE TITLE
Started failing after todays patch

### ISSUE THREAD

**Issue by douglasg14b** [blaxxun-boop/Jewelcrafting#102]
![image](https://github.com/user-attachments/assets/4a8f896d-e83b-4181-8fbf-2bbbed5bac8e)

```
[Warning:  HarmonyX] AccessTools.DeclaredMethod: Could not find method for type ItemDrop+ItemData and name GetTooltip and parameters (ItemDrop+ItemData, int, bool, float)
[Error  : Unity Log] ArgumentException: Undefined target method for patch method static void Jewelcrafting.GemStones+DisplayEffectsOnGems::Postfix(ItemDrop+ItemData item, String& __result)
Stack trace:
HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
Rethrow as HarmonyException: Patching exception in method null
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
Jewelcrafting.Jewelcrafting.Awake () (at <bbb75eb1410048dc832fbcf444815a0f>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.GameObject:.cctor()
PlatformTools.Common.SingletonMonoBehaviour`1:get_Instance()
PlatformTools.PlatformManagerInitializer:OnRuntimeMethodLoad()+
```

**Comment by douglasg14b** [comment]
@blaxxun-boop  Auto closed? 👀

**Comment by douglasg14b** [comment]
Oh I see #101 

Though, unmerged.

**Comment by blaxxun-boop** [comment]
Please do not report issues with mods on patch day. Just assume everything is broken and the mod author is well aware of that. If you have 10 people reach out for all of your 100 mods, it's just spam that delays the work on fixing the mods.

It's generally better to just give the mod author a week or so and if there are still not updates for their mods, you can ask, if the mod is still maintained.

**Comment by douglasg14b** [comment]
I apologize for the issue, I did check for issues, but I did not check for PRs.

<hr>

That said, neither I nor others are always aware of the preferences you as the author may have for your repos. 

I prefer to be notified of breaking changes with my software as soon as it breaks, and know many others in the same bucket. It's not a safe assumption in most cases to assume that "They know", the [bystander effect ](https://en.wikipedia.org/wiki/Bystander_effect) often results in no one saying anything even if there is a problem that is going unnoticed.

I will avoid commenting on _your_ projects in this manner, as you have communicated that preference here. However, I wouldn't expect others to hunt this thread down to gain the same understanding. A readme banner may prevent unwanted bug reports?
